src/WebJobs.Script/Description/ApiHubBindingMetadata.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.IO;
+using System.Threading;
+using Microsoft.Azure.WebJobs.Extensions.ApiHub;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    public class ApiHubBindingMetadata : BindingMetadata
+    {
+        [AllowNameResolution]
+        public string ConnectionString { get; set; }
```

Comment:
Remove this property and use the inherited Connection property on base, following the pattern of the other bindings